gecoscc-installer.sh

```diff
@@ -317,10 +327,12 @@ USER)
 
 PRINTERS)
     echo "LOADING PRINTERS CATALOG"
+    source /opt/rh/python27/enable
```

Comment:
Now it makes sense.

Changed in the way you said, checking if  python 2.7 is installed and then, executing the `source...`.